scss/_menu.scss

```
@@ -35,3 +35,91 @@
         }
     }
 }
+
+.menu {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    visibility: hidden;
+
+    &.show{
+        visibility: visible;
+    }
+
+    &-branding, &-nav {
+        display: flex;
+        flex-flow: column wrap;
+        align-items: center;
+        justify-content: center;
+        float: left;
+        width: 50%;
+        height: 100vh;
+        overflow: hidden;
+
+    }
+
+    &-nav {
+        margin:0;
+        padding: 0;
+        background: transparentize(darken($primary-color, 5),0.1);
+        list-style: none;
+        transform: translate3d(0, -100%, 0);
+        @include easeOut;
+
+        &.show {
+            transform:translate3d(0,0,0);
+        }
+    }
+
+    &-branding {
+        background: transparentize($primary-color, 0.1);
+        transform: translate3d(0, 100%, 0);
+        @include easeOut;
+
+        &.show {
+            transform:translate3d(0,0,0);
+        }
+
+        .portrait {
+            width: 250px;
+            height: 250px;
+            background: url(../img/profile.jpg);
+            background-size:cover;
+            border-radius: 20%;
+            border: solid 3px $secondary-color;
+            opacity:1;
+        }
+    }
+
+    .nav-item {
+        transform: translate3d(600px,0,0);
```

Capaz podes usar la versión más corta si solo vas a mover el elemento en el eje X

```suggestion
        transform: translateX(600px);
```